cmake/Spt3gConfig.cmake.in

```diff
@@ -1,14 +1,28 @@
+@PACKAGE_INIT@
+
 set(SPT3G_SOURCE_DIR @CMAKE_SOURCE_DIR@)
 set(SPT3G_BUILD_DIR @CMAKE_BINARY_DIR@)
+set(SPT3G_INCLUDE_INSTALL_DIR @PACKAGE_INCLUDE_INSTALL_DIR@)
+
+if("@PACKAGE_CMAKE_INSTALL_DIR@") # installed
+	set(SPT3G_INCLUDE_DIRS "@PACKAGE_INCLUDE_INSTALL_DIR@/include/core")
+	foreach(lib @SPT3G_LIBRARIES@)
+		set(SPT3G_LIBRARIES ${SPT3G_LIBRARIES} spt3g::${lib})
+		set(SPT3G_INCLUDE_DIRS "@PACKAGE_INCLUDE_INSTALL_DIR@/${lib}")
```

Comment:
Is `SPT3G_INCLUDE_DIRS` supposed to be a growing list of directories?  Looks like this might be a bug that's propagated from before.

Reply:
I think it is; in general `find_package(X)` produces variables `X_INCLUDE_DIRS` and `X_LIBRARIES` which downstream users can feed into `target_include_directories` and `target_link_libraries`, basically to set the appropriate `-I` and `-l` flags for building things which depend on X.

Reply:
Yeah, my point is that that line should probably be this I think (currently missing the original list variable to append to):

```
set(SPT3G_INCLUDE_DIRS ${SPT3G_INCLUDE_DIRS} …)
```

Reply:
Oh, yes! I will fix that. Sorry; got distracted from the trees by the forest.

Reply:
This should now be fixed, as is the test for whether the installed or uninstalled mode should be used.